learn/core_concepts/relevancy.md

```diff
@@ -102,29 +106,19 @@ The `word position` rule sorts the results by increasing matching word's index n
 
 ::::
 
-## Order of the rules
-
-By default, the built-in rules are executed in the following order to meet most standard needs.
-
-```json
-["typo", "words", "proximity", "attribute", "wordsPosition", "exactness"]
-```
-
-Depending on your needs, you might want to change this order of importance. To do so, you can use the [settings route](/reference/api/ranking_rules.md#update-ranking-rules) of your index.
-
-## Adding your rules
+### Custom rules
 
-New rules can be added to the existing list at any time and placed anywhere in the sequence.
+For now, MeiliSearch supports two custom rules that can be added to [the ranking rules array](#behavior): one for ascending sort and one for descending sort.
 
-A custom rule allows you to create an ascending or descending sorting rule on a given attribute. The attribute **must have a numeric value** in the documents. If any value is not a numeric type, the sorting rule won't be applied. Only numbers can be arranged in ascending or descending order.
+To add a custom ranking rule, you have to communicate either `asc` for ascending order or `desc` for descending order followed by the field name in parentheses.
 
-To add your own ranking rule, you have to communicate either `asc` for ascending order or `desc` for descending order followed by the field name between round brackets.
+- To apply an **ascending sort** (results sorted by increasing value of the attribute): `asc(attribute_name)`
 
-- To apply an **ascending sorting** (results sorted by increasing value of the attribute): `asc(attribute_name)`
+- To apply a **descending sort** (results sorted by decreasing value of the attribute): `desc(attribute_name)`
 
-- To apply a **descending sorting** (results sorted by decreasing value of the attribute): `desc(attribute_name)`
+**The attribute must have a numeric value** in all of the documents contained in that index. **If any value is not a numeric type, the sorting rule won't be applied**.
 
-Add this rule to the existing list of ranking rules using the [settings route](/reference/api/ranking_rules.md#update-ranking-rules).
+Add this rule to the existing list of ranking rules using the [update ranking rules endpoint](/reference/api/ranking_rules.md#update-ranking-rules).
```

Comment:
My current rule of thumb is to link to the API reference with a description of what the endpoint accomplishes (e.g. `the add documents endpoint`, or `the update ranking rules endpoint` above). Generally the text in the link is the same as the anchor I'm linking to within the API reference.

Reply:
Ah, yes. Since I'd rather avoid using HTTP verbs or full routes (intimidating for newbies), maybe our solution is just to put the endpoint name (same as its section/header name in the API reference) in a code snippet.

Some examples:
- The `add documents` endpoint
- The `update ranking rules` endpoint

One possible downside is that the styling for code snippets and links do not always play well together—but this should be easily fixed with CSS when we move to Docusaurus.